ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js

```diff
@@ -305,27 +278,13 @@
       'locale': 'ja',
       'missingTranslations': true,
       'replaceSync': {
-        'trench yovra': 'トレンチ・ヨヴラ',
```

Comment:
Does this section still need `missingTranslations`?

Reply:
This is missing, on `replaceSync`:
`'Depth Grip': `
`'Doxa': '`
`'Katharsis': `
`'Noesis':` 
`'Sinister Bubble':`

Reply:
```
'Depth Grip': 'ハンド・オブ・オーシャン',
'Doxa': 'ドクサの広間',
'Katharsis': 'カタルシスの広場',
'Noesis': 'ノエシスの間',
```

Reply:
Thank you. I am obliged to leave `missingTranslations:true`, because translations are again missing after `npm run test`.
```
C:\Users\Bauer\Documents\GitHub\cactbot\ui\raidboss\data\05-shb\dungeon\anamnesis_anyder.js:locale ja:no translation for replaceText 'Unknown Add'
C:\Users\Bauer\Documents\GitHub\cactbot\ui\raidboss\data\05-shb\dungeon\anamnesis_anyder.js:locale ja:no translation for replaceText 'Swing/Swipe/Cyclone'
C:\Users\Bauer\Documents\GitHub\cactbot\ui\raidboss\data\05-shb\dungeon\anamnesis_anyder.js:locale ja:no translation for replaceText 'Hammer/Blade Mark'
C:\Users\Bauer\Documents\GitHub\cactbot\ui\raidboss\data\05-shb\dungeon\anamnesis_anyder.js:locale ja:no translation for replaceText 'Blade/Hammer Mark'
C:\Users\Bauer\Documents\GitHub\cactbot\ui\raidboss\data\05-shb\dungeon\anamnesis_anyder.js:locale ja:no translation for replaceText 'Pyre/Hearth'
```

Reply:
Yes these lines should be translated. But I have time on weekend only, so I'll do it later. Thanks for mentioning. (ㆁωㆁ*)